docs/install/2-install-server-with-podman-and-systemd.rst

```diff
@@ -0,0 +1,328 @@
+
+
+Prepare the orchestrator configuration
+######################################
+
+1.  Get configuration file.
+    As of now, the container can not be configured with environment variables, we should use a configuration file, mounted inside of the container.
+    To do this, you can get the current configuration file from the container, edit it, and mount it where it should be in the container.
+    Let's create a file at ``~/.config/inmanta/inmanta.cfg``, we can take as template the default file already packaged in our
+    container image.
+
+    .. only:: oss
+
+        .. code-block:: sh
+
+            mkdir -p ~/.config/inmanta
+            podman run --rm -ti ghcr.io/inmanta/orchestrator:latest cat /etc/inmanta/inmanta.cfg > ~/.config/inmanta/inmanta.cfg
+
+    .. only:: iso
+
+        .. code-block:: sh
+            :substitutions:
+
+            mkdir -p ~/.config/inmanta
+            podman run --rm -ti containers.inmanta.com/containers/service-orchestrator:|version_major| cat /etc/inmanta/inmanta.cfg > ~/.config/inmanta/inmanta.cfg
+
+2.  Update database settings
+    It is very unlikely that your database setup will match the one described in the default config we just got.  Update the configuration in the ``[database]`` section
+    to reflect the setup you have.
+
+.. only:: iso
+
+    3.  Get the license files
+        Together with the access to the inmanta container repo, you should also have received a license and an entitlement files.  The orchestrator will need them
+        in order to run properly.  You can also place them in a config directory on your host.  After this step, we assume that this folder is
+        ``~/.config/inmanta/license/`` and that both files are named ``com.inmanta.license`` and ``com.inmanta.jwe`` respectively.
+
+        .. code-block:: console
+
+            $ tree .config/inmanta
+            .config/inmanta
+            ├── inmanta.cfg
+            └── license
+                ├── com.inmanta.jwe
+                └── com.inmanta.license
+
+            2 directories, 3 files
+
+
+.. _setup-systemd-unit:
+
+Start the server with systemd
+#############################
+
+Here is a systemd unit file that can be used to deploy the server on your machine.
+
+.. only:: oss
+
+    .. code-block:: 
+
+        [Unit]
+        Description=Podman 
+        Documentation=https://docs.inmanta.com
+        Wants=network-online.target
+        After=network-online.target
+        RequiresMountsFor=%t/containers
+
+        [Service]
+        Environment=PODMAN_SYSTEMD_UNIT=%n
+        Restart=on-failure
+        TimeoutStopSec=70
+        ExecStart=/usr/bin/podman run \
+                --cidfile=%t/%n.ctr-id \
+                --cgroups=no-conmon \
+                --sdnotify=conmon \
+                -d \
+                --replace \
+                --publish=127.0.0.1:8888:8888 \
+                --uidmap=993:@%U \
+                --uidmap=0:1:993 \
+                --gidmap=993:@%G \
```

Comment:
The `@%U` and `@%G` syntax is not supported on the podman version available on Rocky Linux 8 and 9 as discussed.

Reply:
Fixed in https://github.com/inmanta/inmanta-core/pull/7025/commits/d4acb57193b6d0a51866e7ca6defe92276bfbaa1